x-pack/plugins/security_solution/common/api/detection_engine/model/rule_schema/rule_request_schema.test.ts

```diff
@@ -1290,10 +1290,29 @@ describe('rules schema', () => {
       expect(getPaths(left(message.errors))).toEqual(['invalid keys "data_view_id"']);
     });
 
-    test('You can optionally send in an array of investigation_fields', () => {
+    test('You can omit investigation_options', () => {
       const payload: RuleCreateProps = {
         ...getCreateRulesSchemaMock(),
-        investigation_fields: ['field1', 'field2'],
+        investigation_options: {
+          fields: ['foo', 'bar'],
+        },
+      };
+      // in case in the future we add "investigation_options"
+      // to "getCreateRulesSchemaMock", want to make clear we're
+      // testing it's absence
+      delete payload.investigation_options;
```

Comment:
I think, in this case, you can omit setting any `investigation_options` in payload, just few lines above

Reply:
Yea, think I was just overthinking it in the case this field was ever added to `getCreateRulesSchemaMock` but probably unnecessary future planning. Will clean up.